cmake/UnitTests.cmake

```diff
@@ -763,6 +764,11 @@ list(APPEND IGPU_OPENCL_FAILED_TESTS "ABM_AddKernel_MultiTypeMultiSize - double"
 list(APPEND IGPU_OPENCL_FAILED_TESTS "hipStreamSemantics") # SEGFAULT
 list(APPEND IGPU_OPENCL_FAILED_TESTS "cuda-simpleCallback") # SEGFAULT
 list(APPEND IGPU_OPENCL_FAILED_TESTS "stream") # SEGFAULT
+#Flaky ones?
+#list(APPEND IGPU_OPENCL_FAILED_TESTS "graphMatrixMultiply") # timeout
```

Comment:
Can't reproduce this one

Reply:
There might be some issues when running multiple tests in parallel, as that test didn't fail when ran separately. But it's difficult to see why it would be a client app issue. Perhaps some race issue problem in the driver when multiple contexts on the same cl platform.